charts/renga/charts/gitlab/templates/_gitlab.rb.tpl

```diff
@@ -13,7 +13,7 @@ GitLab Omnibus configuration
 ##! URL on which GitLab will be reachable.
 ##! For more details on configuring external_url see:
 ##! https://docs.gitlab.com/omnibus/settings/configuration.html#configuring-the-external-url-for-gitlab
-external_url '{{ template "http" . }}://{{ .Values.global.renga.domain }}/gitlab'
+external_url '{{ template "http" . }}://gitlab.{{ .Values.global.renga.domain }}'
```

Comment:
could this just be `.Values.global.renga.gitlab_url`? Then it's just up to the ingress spec and whoever is running the chart can decide where they want the services to run.